python/bindings.cpp

```diff
@@ -52,6 +52,26 @@ py::array asArray(std::vector<std::string>&& values) {
 }
 
 
+// Return a 2D structure from a flat std::vector
+template <typename T>
+py::array asArray(std::vector<T>&& values, ssize_t n_cols) {
```

Comment:
Couldn't this be deleted then?

Reply:
True, not needed for the moment. You think it's not worth to keep, just in case for the future?

Reply:
I guess that's OK, although I feel like it's a little dangerous, still.